mlrun/serving/routers.py

```diff
@@ -621,6 +621,18 @@ def do_event(self, event, *args, **kwargs):
                     if response_body["inputs"] and response_body["outputs"]:
                         break
                 response.body = response_body
+            elif (
+                name == self.name
+                and event.method == "GET"
+                and subpath == OperationTypes.explain.name
```

Comment:
In `V2ModelServer` there is a method called `explain` that should return a string value. Users can override it to enable calling `/explain`. So, I think this is not what Yaron intended for you to do.

```python
    def explain(self, request: Dict) -> Dict:
        """model explain operation"""
        raise NotImplementedError()
```